lib/rules/sotd/mailing-list.js

```diff
@@ -45,8 +45,8 @@ exports.check = function (sr, done) {
         if (!foundRepo) {
```

Comment:
if this is a REC, we ought to require a Repo rather than allowing the use of mailing list. This will align with the maintenance discussion within the AB.